### PR TITLE
Add in Type Model

### PR DESCRIPTION
**Pull request by lamtu1**
Create the model for Pokemon's typing and create an association with it.

### CODE REVIEW

File: app/views/elements/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<p style="color: green"><%= notice %></p>
```
Comment (straleyb) [inline app/views/elements/index.html.erb]: Lets take this style out and into css

File: app/views/elements/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<p style="color: green"><%= notice %></p>
```
Comment (straleyb) [inline app/views/elements/show.html.erb]: same with this one

**Review by straleyb — APPROVED** [review]

File: app/views/elements/index.html.erb (hunk 1 above)
Comment (lamtu1) [reply app/views/elements/index.html.erb]: Added in CSS

File: app/views/elements/show.html.erb (hunk 2 above)
Comment (lamtu1) [reply app/views/elements/show.html.erb]: Same here too

File: app/views/elements/index.html.erb (hunk 1 above)
Comment (straleyb) [inline app/views/elements/index.html.erb]: Missed a `class="notice_identify"`

**Review by straleyb — CHANGES_REQUESTED** [review]

**Pull request merged**